.betterer.results

```diff
@@ -1574,7 +1574,8 @@ exports[`better eslint`] = {
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "12"],
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "13"],
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "14"],
-      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "15"]
+      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "15"],
+      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "16"]
```

Comment:
Additional one here because we put the error message (untranslated) in the `<Alert>` (this can't be translated right now)

Reply:
This is actually because of the `title="Expression failed"` in the component - it's added that to the top and pushed the rest down. This looks like an additional one but is just because of the commas

Reply:
Ah right, it tends to shuffle those around – now if only it also told us what line of code it was 😓